src/TestEnvironment.Docker.Containers.Elasticsearch/TestEnvironment.Docker.Containers.Elasticsearch.csproj

```diff
@@ -2,19 +2,14 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <LangVersion>Latest</LangVersion>
     <AssemblyName>TestEnvironment.Docker.Containers.Elasticsearch</AssemblyName>
     <RootNamespace>TestEnvironment.Docker.Containers.Elasticsearch</RootNamespace>
     <Version>1.0.5</Version>
     <PackageId>TestEnvironment.Docker.Containers.Elasticsearch</PackageId>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Authors>Aliaksei Harshkalep</Authors>
     <Description>Add Elasticsearch container specific functionality.</Description>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <PackageTags>docker tests elasticsearch container</PackageTags>
-    <PackageProjectUrl>https://github.com/Deffiss/testenvironment-docker</PackageProjectUrl>
```

Comment:
Should the staff like that be removed?

Reply:
This value was duplicated across all project files, so i extracted it to Directory.Build.props.